apps/pie-docs/test/system/footer.spec.js

```diff
@@ -1,5 +1,5 @@
 import { test, expect } from '@playwright/test';
-import { disableCookieBanner } from '../helpers/playwright-helper';
+import { disableCookieBanner } from '../playwright/playwright-helper';
 
 test.beforeEach(async ({ page, context }) => {
     await page.goto(process.env.BASE_URL);
```

Comment:
can we destructure baseURL here and follow this approach https://github.com/justeattakeaway/pie/pull/302/files#diff-8583d0dc72f3061f86c337d59987f7630ce698f636d9c249fe42b203620be1d4R5-R6

Reply:
Oh yep, looks like this got missed! Good spot 🎉